web-console/knikubevirt/guest-agent-not-installed/guest-agent-not-installed.md

```diff
@@ -0,0 +1,22 @@
+#Guest Agent not installed
+
+The purpose of this document is to show what a VM would look like without the guest agent installed. 
+
+In the list view we show an “info” icon with the status with a tooltip that show that the guest agent isn’t installed. Further information can be provided in a tooltip.
```

Comment:
If it's really a potential "warning" where the informational icon should be yellow...should we use the "warning" icon instead?

Reply:
Ya I went back and fourth on that too. It seems like it's kind of in the middle of a warning and info though, the user may have intended for this outcome so see warning icons could alarm them unnecessarily. It's open to feedback though :)

Reply:
It's been a while, but IIRC at one point we considered using an orange Running icon to indicate that although the VM is still technically "Running" there's something about it that we recommend addressing. I think Matt or someone suggested adding an asterisk to the end of "Running" too so that we don't rely solely on color. I'm not quite sure an Info icon feels right in this context, but I don't feel strongly about this.

The top of the [VM List doc](http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/vm-list/vm-list) has an example of an orange Running icon that's used to indicate that a running VM needs to be restarted to apply pending changes. Whatever we decide, we should probably update that to use the same icon as well.

![0-0-main](https://user-images.githubusercontent.com/9122899/57719425-be5f8080-764d-11e9-9769-0098e4068516.png)

Reply:
I like the running icon (with colour change). I agree that "running *" would be a good a11y approach.

Reply:
@yfrimanm can you update the in-progress icon to be a orange running icon